app/components/notification-list/template.hbs

```diff
@@ -0,0 +1,27 @@
+<div class="container">
+    <div class="row">
+        <div class="notification-list col-md-8">
+            <div class="notification-list-heading-container">
+                <h4 class="notification-list-heading">{{t 'components.notificationList.heading'}}</h4>
+                <span class="helper-text">
+                    {{t 'components.notificationList.helperText'}}
+                    <a href={{serviceLinks.settings}}>
+                        {{t 'components.notificationList.userSettings'}}
+                    </a>.
+                </span>
+            </div>
+
+            {{#if fetchData.isRunning}}
+                {{loading-indicator classes='ball-dark text-center'}}
```

Comment:
Would you mind using a skeleton screen instead of a the ball scale loading here? We are in the process of phasing those loaders out.